pkgs/applications/networking/cluster/coop-cloud/abra.nix

```diff
@@ -0,0 +1,26 @@
+{buildGoModule, fetchFromGitea, lib}:
```

Comment:
Nit: usual coding style is to leave spaces around {} and [].

`{ buildGoModule, fetchFromGitea, lib }:`